server/lib/util.js

```diff
@@ -182,3 +184,27 @@ function convertStringToNumber(obj) {
   });
   return obj;
 }
+
+/*
+ * rename an object's keys
+ */
+
+function renameKeys(objs, newKeys) {
+  const stringType = 'string';
+  const formatedKeys = (typeof stringType) ? JSON.parse(newKeys) : newKeys;
```

Comment:
Just FYI - this `typeof` code isn't complete.  I think you meant to write `typeof newKeys === stringType`.

Reply:
your're right